shared/src/main/scala/com/nitro/nmesos/config/model.scala

```diff
@@ -38,11 +38,16 @@ object model {
     instances: Option[Int]
   )
 
+  case class PortMap(
+    containerPort: Int,
+    hostPort: Int
+  )
+
   case class Container(
     image: String,
     command: Option[String],
     forcePullImage: Option[Boolean],
-    ports: Option[Seq[Int]],
+    ports: Option[Seq[Either[Int, PortMap]]],
```

Comment:
Instead of a `Option[Seq[Either[Int, PortMap]]]`, I'd like to see here just a `Option[Seq[PortMap]]` and move the optional `host port` as an optional inside the case class.
Something like this represent better what we want to have here:
```case class PortMap( containerPort: Int,   hostPort: Optional[Int]  )```

Reply:
Olso, if you go for it, the format will be something like:
```
convertTo[String].split(":").map(_.toInt).List match {
case containerPort :: Nil =>  PortMap(containerPort, None)
case containerPort :: hostPort :: Nil =>  PortMap(containerPort, Some(hostPort)
}
```